app/views/feedback.jade

```diff
@@ -8,7 +8,8 @@ html
 
     link(rel='stylesheet' href='styles/main.css')
     script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
-    script(type="text/javascript" src="/scripts/satisfaction.js")
+    script(type="text/javascript" src="https://code.jquery.com/jquery-2.2.3.min.js")
+    script(type="text/javascript" src="/scripts/satisfaction.min.js")
```

Comment:
Okay. IMO you are complicating your build process to support a one off script. This script doesn't need any of the transformations that browserify is doing. It doesn't import or export any modules, use ES6 synax, or even warrant minification. Why not just put this script in a directory outside of the one the browserify task references, and just include it in this file? Then your browserify task doesn't have to do twice as much as work.

As far the location of scripts head vs. body, you should just try and be consistent. Technically putting it in the body will help eliminate the browser locking briefly as it parses your scripts before rendering the body.